articles/overview/what-is-quantum-computing.md

```diff
@@ -44,11 +44,16 @@ Using quantum programs to model quantum systems themselves is another applicatio
 
```

Comment:
Colon is odd and doesn't make sense here:
and the **no cloning** property of qubits,[comma, not a colon] meaning the inability for qubits to be copied without detection.

Reply:
Fixed